aten/src/ATen/native/native_functions.yaml

```diff
@@ -6299,18 +6293,8 @@
   device_check: NoCheck
   dispatch: {}
 
-- func: _nested_get_min_seqlen(Tensor self) -> Tensor
-  variants: function
-  device_check: NoCheck
-  dispatch: {}
-
-- func: _nested_get_max_seqlen(Tensor self) -> Tensor
-  variants: function
-  device_check: NoCheck
-  dispatch: {}
-
-- func: _nested_get_jagged_dummy(Tensor any) -> Tensor
-  category_override: dummy
+- func: _nested_get_jagged_metadata(Tensor any) -> Tensor
+  category_override: dummy  # Is the "dummy" str special-cased
```

Comment:
what does this do again?

Reply:
From reading `torchgen/api/python.py`, it looks like (1) certain functions based on their name (ending with _like, starting with new_) or based on signature (does not take tensor as input, but returns one as output) are implicitly categorized as factories. Factory-like functions ordinarily  have TensorOptions implicitly added, but it seems that dummy can be used to override that and say no I don't actually want TensorOptions. 

I'm confused though because I can't see why this would be categorized as a factory. Maybe it didn't take Tensor as input previously? I will try removing it.